cypress/integration/cbl/admin/tasks-manager.js

```diff
@@ -323,30 +323,31 @@ describe('CBL / Admin / Tasks Manager', () => {
             .within(() => {
 
                 // click cloned task selector
-                cy.extGet('slate-cbl-taskselector[name=ClonedTaskID]')
-                    .should('exist')
-                    .within(() => {
-
-                        cy.root()
-                            .get('.x-form-arrow-trigger')
-                            .click()
-
-                    });
+                cy.get('input[name="ClonedTaskID"]')
+                    .should('have.length', 1)
+                    .parents('.x-form-trigger-wrap')
+                        .should('have.length', 1)
+                        .within(() => {
+                            cy.get('.x-form-arrow-trigger')
+                                .click()
+                        });
```

Comment:
There's no reason to use `.within()` if you're not using the scope more than once. This could just be:

```suggestion
                        .find('.x-form-arrow-trigger')
                            .click();
```